### PR TITLE
OSDOCS-10165: 4.14.20 RN

### PR DESCRIPTION
**Pull request by obrown1205**
Version(s):
4.14

Issue:
[OSDOCS-10165](https://issues.redhat.com/browse/OSDOCS-10165)

Link to docs preview:
https://74332--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes.html

QE review:
N/A

Additional information:

### CODE REVIEW

**Review by xenolinux — COMMENTED** [review]
/remove-label peer-review-needed
/remove-label peer-review-in-progress
/label peer-review-done

LGTM

**Comment by obrown1205** [comment]
/remove-label peer-review-needed
/remove-label peer-review-in-progress
/label peer-review-done

LGTM

File: release_notes/ocp-4-14-release-notes.adoc [hunk 1]
```diff
@@ -3288,6 +3288,25 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+[id="ocp-4-14-20"]
+=== RHBA-2024:1564 - {product-title} 4.14.20 bug fix update and security update
```
Comment (stevsmit) [inline release_notes/ocp-4-14-release-notes.adoc:3292]: RHBA-2024:1564 here, but RHSA-2024:1681 in the body.

**Pull request merged**